### ISSUE TITLE
Document configDir (Hugo 0.53)

### ISSUE THREAD

**Issue by bep** [gohugoio/hugoDocs#686]
https://github.com/gohugoio/hugo/pull/5501

* configDir
* environment
* Replace menu with menus, but keep a reference to the old (both works)

**Comment by regisphilibert** [comment]
Question can the user use both `/config.yaml` AND `/config/` directory structure ? At the moment it seems `/config.yaml` always wins.

**Comment by bep** [comment]
Yes, you can have both, but I don't see why you would. I don't remember the "precedence rules" for this, but I assume they make sense ...

**Comment by regisphilibert** [comment]
In addition to updating [Getting Started/Directory Structure](https://gohugo.io/getting-started/directory-structure/) and [Getting Started/Configuration](https://gohugo.io/getting-started/directory-structure/), I think we should also add a new page to detail the new environment system of hugo, (`--environment` flag, `site.Environment` etc...)

I'm thinking about adding this page underneath __Hosting & Deployement__

Does anybody see any reason not to ? @budparr @kaushalmodi @bep

**Comment by regisphilibert** [comment]
Actually there's a lot of tutorials in there. Not sure we need a page to explain environments yet as long as `hugo.Environment` and the cli tag is documented.
